make/photon/clair-adapter/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM photon:2.0
+
+RUN tdnf install -y sudo >>/dev/null\
+    && tdnf clean all \
+    && mkdir /clair-adapter/ \
+    && groupadd -r -g 10000 clair-adapter \
+    && useradd --no-log-init -m -r -g 10000 -u 10000 clair-adapter
+
+COPY ./make/photon/clair-adapter/binary/harbor-scanner-clair /clair-adapter/clair-adapter
+
+RUN chown -R 10000:10000 /clair-adapter \
+    && chmod u+x /clair-adapter/clair-adapter
+
+EXPOSE 8080
+
+HEALTHCHECK --interval=30s --timeout=10s --retries=3 CMD curl -sS 127.0.0.1:8080/healthy || exit 1
```

Comment:
You should use `/metadata` API endpoint to do check. No healthy endpoint.

Reply:
there is `/healthy` in the clair adapter 
```go
probeRouter.Methods(http.MethodGet).Path("/healthy").HandlerFunc(handler.GetHealthy)
```

`/metadata` maybe need authorization if the adapter implemented the auth.

Reply:
It is not defined in the open API spec

Reply:
Yes, it is not in the open API spec, this is the image of clair adapter and `/healthy` is defined by clair adapter i think it is ok to use this for the health checking.

When there is another adapter eg named `foo` maybe its health checking endpoint is `/foo` and then the `HEALTHCHECK` in its docker image maybe `HEALTHCHECK --interval=30s --timeout=10s --retries=3 CMD curl -sS 127.0.0.1:8080/foo || exit 1`

This image care the clair adapter itself, not care the open API spect defined by harbor with other scanners.

Reply:
Ok, but we need to make sure the `healthy` check endpoint will be always there.